kqueen/kubeapi.py

```diff
@@ -51,6 +51,8 @@ def get_api_client(self):
         # functions in https://github.com/kubernetes-client/python
         client_config = type.__call__(client.Configuration)
         kubeconfig = self.cluster.get_kubeconfig()
+        if kubeconfig is None:
```

Comment:
maybe resolbve it with try catch block in self.cluster.get_kubeconfig method? imo it should be better

Reply:
get_kubeconfig just returns kubeconfig, it should not raise any error

Reply:
in what case it can be None? get_kubeconfig is really important for 70% kqueen actions, so why it can be None?

Reply:
If cluster has failed state

Reply:
ok, agree